test/e2e/proxy_test.go

```diff
@@ -61,6 +61,13 @@ func TestOAuthProxyE2E(t *testing.T) {
 			},
 			pageResult: "URI: /",
 		},
+		"basic-with-timeout": {
+			oauthProxyArgs: []string{
+				"--upstream=http://localhost:8080",
+				"--upstream-timeout=30s",
```

Comment:
isn't this the default that's actually set in all the other tests?

Reply:
Yes. However, I added it only to get the end-to-end test to verify that the flag is working.

I will drop it. Given that this is not that useful.

Reply:
Done. Thank you!